testv2/tests.yml

```diff
@@ -312,6 +312,42 @@
   infrastructures:
     - name: aws_defaults
 
+- scenario: csi_ccm_migration
+  initVersion: v1.21.14
+  infrastructures:
+    - name: aws_defaults 
+    - name: openstack_default
+    - name: azure_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
```

Comment:
We can probably drop 1.21 tests because 1.21 reached EOL a while ago.

Reply:
Done, removed that stuff